code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -370,6 +370,9 @@
 				throw_at(throw_target, 200, 4)
 				damage_clothes(400 - bomb_armor, BRUTE, "bomb")
 			else
+				for(var/I in contents)
+					var/atom/A = I
+					A.ex_act(EXPLODE_DEVASTATE)

```

Comment:
use the `severity` var so the strength is the same across all explosions

Reply:
I explicitly wanted to avoid that outcome so that when someone welderbombs during a blob you aren't left arguing with an admin as to why you're not culpable for bombing the entire crew. 

I think we even explicitly made this choice before so that light explosions weren't deleting a bunch of shit in your bag. If the ex_act code were better then sure but I only wanted to reserve this outcome for cases where the bag is destroyed.